analytics/analytics-ui/src/app/analytics/analytics.component.html

```diff
@@ -1,8 +1,8 @@
 <div style="margin-left:110px">
   <h1>Analytics</h1>
 </div>
-<mat-card style="min-height: 300px;">
-  <mat-grid-list cols="4">
+<mat-card>
+  <mat-grid-list cols="4" gutterSize="20px">
```

Comment:
Is this going to cause an issue when it comes to different sized screens?

Reply:
Now using vw instead of px to automatically scale the gutter.